Pages/Crime.cshtml

```diff
@@ -5,9 +5,10 @@
 }
 
 <script>
-    function myFunction() {
+    function filterZipCode() {
```

Comment:
changed function name

Reply:
http://www.j-io.org/Javascript-Naming_Conventions/#:~:text=%20JavaScript%20Naming%20Conventions%20%201%20Essential%20Conventions%0AMinimizing,Comments%2013%20Documentation%2014%20Additional%20Readings%20More%20

Reply:
Good.  New name is more readable.